pkg/comp-functions/functions/common/namespace-permissions.go

```diff
@@ -0,0 +1,72 @@
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/vshn/appcat/v4/pkg/common/utils"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const roleBindingName = "appcat:services:read"
+
+func GetOrg(instance string, svc *runtime.ServiceRuntime) string {
+	ns := &corev1.Namespace{}
+
+	err := svc.GetObservedKubeObject(ns, instance+"-claim-ns-observer")
```

Comment:
I don't like the fact that we are hard coding parts of the object names. There's no other way to get the namespace?

Reply:
I've moved this into a const. Is it accetable for you now?

Reply:
I meant something else but it's fine. We need a better way to managed names in our comp functions, pre pending suffixes and prefixes all over is gonna cost us dearly later on.